File: hw/pazmino/u3/HW37StrategyExercise/HW38StrategySortApp/src/main/java/ec/edu/espe/hw38strategysortapp/view/SortApp.java
package ec.edu.espe.hw38strategysortapp.view;

import ec.edu.espe.hw38strategysortapp.controller.SortingContext;
import java.util.Arrays;

/**
 *
 * @author Brandon Pazmino
 */
public class SortApp {

    public static void main(String[] args) {
        int data[] = {3, 6, 4, 6, 7, 8, 5, 6, 7, 5, 3, 3};
        SortingContext sc = new SortingContext();
        int sortedList[] = sc.sort(data);

        System.out.println("Original array: " + Arrays.toString(data));
        System.out.println("Sorted array:   " + Arrays.toString(sortedList));

    }
}

File: hw/pazmino/u3/HW37StrategyExercise/hw38StrategySortAppPython/view/sort_app.py
from controller.sorting_context import SortingContext

def main():
    data = [3, 6, 4, 6, 7, 8, 5, 6, 7, 5, 3, 3]
    
    context = SortingContext()
    
    sorted_data = context.sort(data.copy())
    
    print(f"Original array: {data}")
    print(f"Sorted array:   {sorted_data}")

if __name__ == "__main__":
    main()
